### PR TITLE
Feature/dom

### PR DESCRIPTION
**Pull request by hlee92**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -165,4 +165,7 @@ <h2 class="is-size-4">July 25, 1999 in Lagos, Nigeria</h2>
     </footer>
     <script src="./assets/js/script.js"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (thevcr) [inline index.html:175]: We can probably remove the extra spacing at the end

**Comment by abel-baker** [comment]
🙌 Weather feature element creation and display works great, looks great

`whenAndWhereContainerEl.innerHTML` is being set with unstyled text on line 58.  Do we want to replace it with this from your earlier commit?

```
const locationH2 = document.createElement("h2");
locationH2.setAttribute("class", "is-size-4");
locationH2.textContent = `${date} in ${fetchData.locations[address].name}`;
whenAndWhereEl.appendChild(locationH2);
```

File: assets/js/script.js [hunk 2]
```diff
@@ -1,51 +1,111 @@
...
+
+var displayWeatherData = function (fetchData, date, address) {
+  // check if api returned any repos
+  if (!fetchData) {
+    whenAndWhereContainerEl.textContent = "No weather data found. Update your search parameters and please try again.";
+    return;
+  }
+
+  console.log(fetchData);
+  whenAndWhereContainerEl.innerHTML = `${date} in ${fetchData.locations[address].name}`;
```
Comment (abel-baker) [inline assets/js/script.js]: 🙌 Weather feature element creation and display works great, looks great

`whenAndWhereContainerEl.innerHTML` is being set with unstyled text on line 58.  Do we want to replace it with this from your earlier commit?

```
const locationH2 = document.createElement("h2");
locationH2.setAttribute("class", "is-size-4");
locationH2.textContent = `${date} in ${fetchData.locations[address].name}`;
whenAndWhereEl.appendChild(locationH2);
```

Comment (thevcr) [reply assets/js/script.js]: +1 I did that more as a placeholder initially so feel free to remove.

**Review by thevcr — CHANGES_REQUESTED** [review]

**Pull request merged**